server/command-line/storage.ts

```diff
@@ -65,4 +78,46 @@ function isUserLogEnabled(manager: ClientManager, user: string): boolean {
 	return conf.log;
 }
 
+async function runCleaning(user: string) {
```

Comment:
let's make user optional then

```suggestion
async function runCleaning(user?: string) {
```